src/main/java/uk/gov/ch/repository/update/trusts/TrustLinksRepository.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.ch.repository.update.trusts;
+
+import java.util.List;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.gov.ch.model.update.trusts.TrustLinkData;
+
+public interface TrustLinksRepository extends
+        PagingAndSortingRepository<TrustLinkData, Long> {
+
+    @Query(value =
```

Comment:
alternatively
"select t.trust_id, ca.corporate_body_appointment_id
from corporate_body cb
join corporate_body_appointment ca on ca.corporate_body_id = cb.corporate_body_id
join corbod_appt_trust_link cat on ca.corporate_body_appointment_id = cat.corporate_body_appointment_id
join trust t on t.trust_id = cat.trust_id
where cb.incorporation_number = ?"
[added that alternative to the ODP]